var/spack/repos/builtin/packages/bolt/package.py

```diff
@@ -19,17 +19,23 @@ class Bolt(CMakePackage):
 
     homepage = "http://www.bolt-omp.org/"
     url      = "https://github.com/pmodels/bolt/releases/download/v1.0b1/bolt-1.0b1.tar.gz"
+    git      = "https://github.com/pmodels/bolt.git"
 
+    version("develop", branch="master")
+    version("1.0rc1", "77733ba2ad9440c29b36d1c1411a9793")
     version("1.0b1", "df76beb3a7f13ae2dcaf9ab099eea87b")
 
     depends_on('autoconf', type='build')
+    depends_on('argobots')
     depends_on('automake', type='build')
+    depends_on('cmake', type='build')
```

Comment:
This line isn't actually needed, the `CMakePackage` base class already adds it automatically.
```suggestion
```